### DISCUSSION TITLE
About Nginx + Frp cause a bug to happen

### DISCUSSION THREAD

**Discussion by Cheng0x** [Chevereto/Chevereto-Free#158] [Help needed]
I used nginx and [FRP](https://github.com/fatedier/frp) to publish chevereto, but the CSS resources were lost. See code for details
----
![image](https://user-images.githubusercontent.com/79407037/150816523-32d51a8e-9032-4e3b-9412-9eb8e377c681.png)
----
![image](https://user-images.githubusercontent.com/79407037/150816823-4adda32b-e0c2-4b95-a57d-fa27f117602a.png)
----
### nginx
![image](https://user-images.githubusercontent.com/79407037/150817002-e1aaf52a-5241-4e7e-a4bc-19df9fba78e6.png)

### frp
![image](https://user-images.githubusercontent.com/79407037/150817298-19ca644d-53d6-4e48-b576-f589e0e66a39.png)
----
![image](https://user-images.githubusercontent.com/79407037/150818442-791c3f06-f704-4fed-9fa1-e27d8f82b55c.png)

**Comment by rodber** [comment]
The URL in Chevereto-Free is auto detected, here: https://github.com/rodber/chevereto-free/blob/1.6/app/loader.php#L93

It uses the value of `$_SERVER['HTTP_HOST']` to determine the hostname, so you either pass the server var or you inject directly your hostname in the loader file.

**Reply by Cheng0x** [reply]
Ths，I hava solved.
